pkg/csi/service/common/constants.go

```diff
@@ -248,4 +248,7 @@ const (
 	TriggerCsiFullSync = "trigger-csi-fullsync"
 	// CSIVolumeManagerIdempotency is the feature flag for idempotency handling in CSI volume manager
 	CSIVolumeManagerIdempotency = "csi-volume-manager-idempotency"
+	// RemoveVCCredsInNodes is the feature flag used for topology feature
+	// to avoid taking in VC credentials in node daemonset
+	RemoveVCCredsInNodes = "node-topology-without-vc-creds"
```

Comment:
If we can keep the variable name for the feature flag similar to what we have in the YAML file, it will be better.

Reply:
My bad, done.